packages/core/admin/admin/src/content-manager/pages/App/LeftMenu/index.js

```diff
@@ -8,23 +8,27 @@ import React, { useMemo, useState } from 'react';
 import { useSelector, shallowEqual } from 'react-redux';
 import { useIntl } from 'react-intl';
 import matchSorter from 'match-sorter';
+import { NavLink } from 'react-router-dom';
```

Comment:
Is that a mistake?

Reply:
No, I was just reordering the imports in a way that made sense to me. I can move it back if needed?